docs/system/angel_system_plus_speech_QA.rst

```diff
@@ -0,0 +1,79 @@
+Running Angel System with Speech QA
+======================
+The following are the instructions to run the angel system to track tasks whilst also interacting with GPT-4 using your voice for Question Answering (QA).
+
+At a high level, you must:
+
+1) start up the Angel ARUI on your HoloLens
+2) start the tmuxinator script which runs the angel system and the QA nodes, for example for medical task R18:
+
+.. code-block:: bash
+
+   tmuxinator start demos/medical/Kitware-R18-qa
+
+Note that for the QA portion, you'll need an OpenAI API key, which you export in the tmuxinator QA window after tmuxinator starts.
+
+.. code-block:: bash
+
+   export OPENAI_API_KEY=your_openapi_key
+   export OPENAI_ORG_ID=your-openai-org-id
+
+3) Run the Angel System ASR server. Installation and running are described below.
+
+Angel System ASR Server
+======================
+
+Git-cloning the separate repo
+---------------------
+For now, this repo lives separately from the angel_system. Clone it by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:ColumbiaNLP/angel-system-speech-processor.git
+
+Installing dependencies with apt
+----------------------
+
+.. code-block:: bash
+
+   sudo apt update && apt install -y sox ffmpeg
+
+Running the Server
+----------------------
+
+Create conda environment
+----------------------
```

Comment:
Was it intended for the "Create conda environment" section to be a child of "Running the Server"? If so, I recommend following the section "tiering" documentation [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections). Following that documentation, this section should use `^` characters.